modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/seedlist/Alias.scala

```diff
@@ -0,0 +1,18 @@
+package org.tessellation.node.shared.domain.seedlist
+
+import cats.syntax.all._
+
+import derevo.cats.eqv
+import derevo.derive
+import fs2.data.csv.CellDecoder
+
+@derive(eqv)
+case class Alias(value: String)
+
+object Alias {
+
+  implicit val maybeAliasCellDecoder: CellDecoder[Option[Alias]] = CellDecoder.stringDecoder.map { s =>
+    if ("".equals(s)) none else Alias(s).some
```

Comment:
_.trim.nonEmpty

Reply:
:+1:

Only if `" "` or `"       "` are not the correct aliases (I assume these are not the correct ones). Also `Option.when/unless` can be used to get rid of explicit `none` return.

Reply:
Changed to `Option.when(s.trim.nonEmpty)(Alias(s))`